bin/prettier.js

```diff
@@ -36,13 +36,15 @@ const argv = minimist(process.argv.slice(2), {
   alias: { help: "h", version: "v", "list-different": "l" },
   unknown: param => {
     if (param.startsWith("-")) {
+      // eslint-disable-next-line no-console
```

Comment:
Moving the disabling comment on top of the aforementioned files will not prevent any new wrong console insertion 😄 .